micromouse/Motor.cpp

```diff
@@ -42,29 +123,51 @@ namespace Micromouse
 		{
 			digitalWrite(fwdPin, HIGH);
 			digitalWrite(bwdPin, LOW);
-			analogWrite(pwmPin, (int)(255 * speed));
+			setVoltage((int)(255 * speed), true);
```

Comment:
I decided that speed would refer to a float between -1 and 1, where voltage refers to an int between 0 and 255. That's why setVoltage() takes an int. It doesn't really matter either way though.